app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def index
+        render_paginated rents, each_serializer: RentSerializer, status: :ok
+      end
+
+      def create
+        if new_rent.save
+          render json: new_rent, serializer: RentSerializer, status: :created
+        else
+          render json: { errors: new_rent.errors }, status: :unprocessable_entity
+        end
+      end
+
+      private
+
+      def new_rent
+        @new_rent ||= user.rents.new(create_rent_params)
```

Comment:
No need to use memorizaiton here. I'd write `user.rents.new(create_rent_params)` above in the create method.

Reply:
I would do this in the create method